api/src/main/java/io/opentelemetry/trace/SemanticAttributes.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+/**
+ * Defines constants for all attribute names defined in the OpenTelemetry Semantic Conventions
+ * specifications.
+ *
+ * @see <a
+ *     href="https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md">Semantic
+ *     Conventions</a>
+ */
+public final class SemanticAttributes {
+
+  /** Transport protocol used. */
+  public static final String NET_TRANSPORT = "net.transport";
+  /** Remote address of the peer (dotted decimal for IPv4 or RFC5952 for IPv6). */
+  public static final String NET_PEER_IP = "net.peer.ip";
+  /** Remote port number as an integer. E.g., 80. */
+  public static final String NET_PEER_PORT = "net.peer.port";
+  /** Remote hostname or similar. */
+  public static final String NET_PEER_NAME = "net.peer.name";
+  /** Like net.peer.ip but for the host IP. Useful in case of a multi-IP host. */
+  public static final String NET_HOST_IP = "net.host.ip";
+  /** Like net.peer.port but for the host port. */
+  public static final String NET_HOST_PORT = "net.host.port";
+  /** Local hostname or similar. */
+  public static final String NET_HOST_NAME = "net.host.name";
+  /** Username or client_id extracted from the access token or Authorization header. */
+  public static final String ENDUSER_ID = "enduser.id";
+  /** Actual/assumed role the client is making the request under. */
+  public static final String ENDUSER_ROLE = "enduser.role";
+  /** Scopes or granted authorities the client currently possesses. */
+  public static final String ENDUSER_SCOPE = "enduser.scope";
```

Comment:
Please use full descriptions from the spec.

Reply:
Fixed.